app/views/books/show.html.erb

```diff
@@ -0,0 +1,30 @@
+<h3>Book Title: <%= @book.title %></h3><br>
+
+<% @book.reviews.each do |review|%>
+<ul>
+  <li>
+    Reviewer: <%= review.user.name %><br>
+    Review: <%= review.body %><br>
+    Rating: <%= review.rating %>
+  </li>
+</ul>
+<% end %>
+<br>
+
+<p>Average Rating: <%= Review.average_rating %></p>
+<h3>Highest Rating: <%= Review.highest_rating %></h3>
```

Comment:
this should be scoped to this specific book and right now, this is getting the average, highest, lowest, etc. for all books. We also want to avoid having direct database calls from the view like `Review`...would want to use an instance variable and pass the information to the view.